test/data/validator.output.yaml

```diff
@@ -1,5 +1,8 @@
 jobs:
   main:
+  - annotations:
```

Comment:
I'm kinda confused. Shouldn't this be in the same level as `settings` and `environments`?